link-verifier/verify-links.py

```diff
@@ -393,7 +399,10 @@ def main():
                 os.remove(f)
 
     for link in link_set:
-        is_broken, status_code = test_url(link)
+        if ( ( link[-1] == "/" ) or ( link[-1] == "," ) ):
+            is_broken, status_code = test_url(link[:-1])
+        else:
+            is_broken, status_code = test_url(link)
         if is_broken:
             broken_links.append(link)
```

Comment:
Intentionally use the version of the URL with the coma or slash for the error list. This way the exact link can be searched for in the source file easily.